# This message adds a tag field an object pose
#  Raymond Kirk (Tunstill) Copyright (c) 2020
#  Email: ray.tunstill@gmail.com

string tag # The image tag (used with '*-LABELLED' matching algorithms)
geometry_msgs/Pose pose # The predicted pose